lib/robot.js

```diff
@@ -12,18 +12,34 @@ const logger = require('./logger')
  */
 class Robot {
   constructor ({app, cache, router, catchErrors} = {}) {
-    this.events = new EventEmitter()
+    this.eventHandler = new EventHandler({
```

Comment:
Apologies if this has been mentioned before, but is there a reason you chose to rename `robot.events` to `robot.eventHandler`? I think it's a breaking change for no reason. I myself have used `robot.events.emit()` for a custom, non-webhook event.

Reply:
Ah nevermind, 'twas a dumb question. It seems that this new EventHandler is very much webhook-specific, so its not designed to pass other events.

Reply:
> I myself have used robot.events.emit() for a custom

I’m curious, can you elaborate on your use case for custom events? Can you use create a custom Event Handler to work around it? 

I think ideally `.eventHandler` would not be exposed as public API at all, but it’s not that simple the way it is currently implemented.